packages/ruleset/src/functions/operation-id-naming-convention.js

```diff
@@ -0,0 +1,143 @@
+const pickBy = require('lodash/pickBy');
```

Comment:
This file is essentially the logic from the legacy rule implementation, with really just cosmetic differences to comply with the spectral rule pattern.